docfx_yaml/extension.py

```diff
@@ -1032,18 +1032,47 @@ def pretty_package_name(package_group):
     return " ".join(capitalized_name)
 
 
+# Check is the current lines conform to markdown header format.
+def is_markdown_header(header_line, prev_line):
+    if "#" in header_line:
+        # Check for proper h1 header formatting, ensure there's more than just
+        # the hashtag character.
+        if header_line[header_line.index("#")+1] == " " and \
```

Comment:
If we're looking for both header and a space, we could change the check to `if "# " in header_line:` to look for a header and a space